tests/test_config.py

```diff
@@ -186,3 +186,45 @@ def test_reload_config(self):
             config.load_from_file(LOCAL_CONFIG)
         finally:
             os.chdir('../..')
+
+    def test_local_base_url(self):
+        from dallinger.utils import get_base_url
+        config = get_config()
+        config.ready = True
+        os.chdir('tests/experiment')
```

Comment:
If you use the `experiment_dir` fixture, it takes care of returning to the cwd for you and declutters the test.